docs/registry/README.md

```diff
@@ -330,3 +331,62 @@ spec:
 Note that more policies should probably need to be configured, e.g., allow auto-registration of EventTypes received 
 from within the cluster (e.g., from a response of a Service running in the cluster) as opposed to external services, and so on. 
 We just enumerate three simple cases here.
+
+## FAQ
+
+Here is a list of frequently asked questions that may help clarify the scope of the Registry.
+
+- Is the Registry meant to be used just for creating Triggers or for also setting up Event Sources?
+
+    It's mainly intended for helping the `Event Consumer` with the `discoverability` use case. Therefore, 
+    it is meant for helping out creating Triggers.
+
+- If I have a simple use case where I'm just setting up an Event Source and my KnService is its Sink (i.e., no Triggers involved), 
+is there a need/use for the Registry?
+
+    In this case, we believe there is no need for the Registry. As you can see in the EventType CRD, there is a mandatory 
+    `broker` field. If you are not sending events to a Broker, then there is no need to use a Registry. 
+    Implementation-wise, we can check whether the Source's sink kind is `Broker`, and if so, then register its EventTypes.   
+
+- Is the Registry meant to be used in a single-user environment where the same person is setting up both the Event Source and 
+the destination Sink?
+
+    We believe is mainly intended for multi-user environment. A `Cluster Configurator` persona is in charge of setting up 
+    the Sources, and `Event Consumers` are the ones that create the Triggers. Having said that, it can also be used in 
+    a single-user environment, but the Registry might not add much value compared to what we have right now in terms of 
+    `discoverability`, but it surely does in terms of, for example, `admission control`. 
+    
+- Does a user need to know which type of environment they're in before they should know if they should look at the Registry? 
+In other words, is a Registry always going to be there and if not under what conditions will it be?
+
+    If there are Sources pointing to Brokers, then there should be a Registry. `Event Consumers` will always be able to 
```

Comment:
Since registry is made of CR entries, there's always a registry, but there just might not be any CR => empty registry?

Reply:
That is the right way of putting it. You are right, you can always `kubectl get eventtypes`, in case there is no CR, then the registry will be empty. But there will always be one.
Will update that. Thanks for the comment!